App/Containers/Contact.tsx

```diff
@@ -107,6 +137,22 @@ class ContactModal extends React.Component<Props> {
   onAdd = () => {
     this.props.addContact()
   }
+
+  navigateToDirectMessageThread = () => {
+    // Could be strange bug where directMessageThreadExists is true but the thread is undefined
```

Comment:
My other comment about getting rid of that redundant selector... That would eliminate this potential bug because the thread as well as if it exists or not would be based on the exact same information.

Reply:
Awesome, that makes sense. Thanks for catching that!